sneakers/channels/googleSpread.py

```diff
@@ -0,0 +1,74 @@
+# Authors: Gabriel Butterick and Bonnie Ishiguro
+
+import time
+import json
+import gspread
+from oauth2client.client import SignedJwtAssertionCredentials
+
+from sneakers.modules import Channel
+
+class GoogleSpread(Channel):
+    description = """\
+        Posts data to Google Spreadsheets.
+    """
+
+    requiredParams = {
+        'sending': {
+            'client_email': '',
+            'private_key': '',
+            'google_sheet': ''
```

Comment:
I think descriptions are probably a good idea here, I'm not 100% sure what all these are.